app/models/concerns/host_searching.rb

```diff
@@ -142,6 +145,17 @@ def filter_by_compliance_score(_filter, operator, score)
       { conditions: "hosts.id IN(#{hosts.to_sql})" }
     end
 
+    def filter_by_failed_rules_severity(_filter, operator, value)
+      values = value.split(',').map(&:strip)
+      profiles = RequestStore.store['scoped_search_context_profiles']
+      raise ScopedSearch::QueryNotSupported if profiles.nil?
+      # FIXME: duplicated on #filter_by_ssg_version
+      where = { rule: { severity: values } }
+      base = ::RuleResult.latest(profiles.compact.first.policy_id).joins(:rule).distinct.select(:host_id)
```

Comment:
I was thinking about better solutions, but I guess this is the best one right now...

Reply:
There's `FIXME:` to remind me to extract this into a method

Reply:
I meant the `compact.first.policy_id` part